code/game/objects/effects/decals/crayon.dm

```diff
@@ -28,3 +28,6 @@
 	if(main)
 		paint_colour = main
 	add_atom_colour(paint_colour, FIXED_COLOUR_PRIORITY)
+
+/obj/effect/decal/cleanable/crayon/NeverShouldHaveComeHere(turf/T)

```

Comment:
nigga wtf is this proc name?

Reply:
no bloody clue, check /obj/effect/decal, im simply overriding it here.

Reply:
wanna hear a funny joke https://github.com/tgstation/tgstation/pull/34121

Reply:
wew lad

Reply:
https://www.youtube.com/watch?v=Lr7CKWxqhtw